app/src/main/java/com/example/sabonit/CartActivity.java

```diff
@@ -98,9 +100,23 @@ public void onProductDeleteClick(View view, int position) {
     }
 
     public void goToOrder(View view) {
-        showHowItWorksDialog();
-//        Intent intent = new Intent(this, OrderActivity.class);
-//        startActivity(intent);
+        //showHowItWorksDialog();
+        if (cart.isCartEmpty())
+        {
+            Context context = getApplicationContext();
+            CharSequence text = "You can't order empty cart";
+            int duration = Toast.LENGTH_LONG;
+
+            Toast toast = Toast.makeText(context, text, duration);
+            toast.show();
+            return;
+        }
+        Account.getCurrentAccount().getConfirmedOrders().addCart(cart);
+        Account.getCurrentAccount().setCart(new Cart());
+        String uid = Account.getCurrentAccount().getUID();
+        db.collection("Accounts").document(uid).set(Account.getCurrentAccount());
+        Intent intent = new Intent(this, OrderActivity.class);
+        startActivity(intent);
```

Comment:
nice, maybe can short all these lines and call once to getCurrentAccount and to helzh from him all the other account attributes

Reply:
Yes you are right we will work on that too
maybe we can create a Utils class for it